frontend/src/components/misc/HideStatisticsBarButton.tsx

```diff
@@ -9,26 +9,25 @@
  * by the Apache License, Version 2.0
  */
 
-import { Component } from 'react';
-import React from 'react';
-import { message } from 'antd';
+import React, { FC } from 'react';
 import { EyeClosedIcon } from '@primer/octicons-react';
-import { Tooltip } from '@redpanda-data/ui';
-export class HideStatisticsBarButton extends Component<{ onClick: () => void }> {
-    handleClick = () => {
-        this.props.onClick();
-        message.info('Statistics bar hidden! You can enable it again in the preferences.', 8);
-    };
+import { Flex, Text, Tooltip, useToast } from '@redpanda-data/ui';
 
-    render() {
-        return (
-            <Tooltip label={<span style={{ whiteSpace: 'nowrap' }}>Hide statistics bar</span>} placement="right" hasArrow>
-                <div className="hideStatsBarButton" onClick={this.handleClick}>
-                    <div style={{ display: 'flex', width: '100%' }}>
-                        <EyeClosedIcon size="medium" />
-                    </div>
-                </div>
-            </Tooltip>
-        );
-    }
+export const HideStatisticsBarButton: FC<{ onClick: Function }> = ({onClick}) => {
+    const toast = useToast()
+    return (
+        <Tooltip label={<Text whiteSpace="nowrap">Hide statistics bar</Text>} placement="right" hasArrow>
+            <div className="hideStatsBarButton" onClick={() => {
```

Comment:
Consider using Box and removing classname so that styles are applied in the Chakra-preferred way

Reply:
This seems like it's not working anyway. I'm a bit confused on what to do here. There is an eye icon when I investigate how it should work. But I think it doesn't and I personally find it a bit intrusive. Should we just get rid of this eye icon at all and keep it in the settings only?
<img width="729" alt="Screenshot 2023-09-29 at 9 38 51" src="https://github.com/redpanda-data/console/assets/1083817/5d7e5293-f9f6-4a35-b351-e5f4b1bb461b">

Reply:
Agreed, it does seem out of place. Let's hear from Moritz

Reply:
Lets remove the entire "hide statistics bar" button.
It was an experiment that stuck around for a while, but it's not being used by people.